src/DIRAC/Core/Tornado/Client/private/TornadoBaseClient.py

```diff
@@ -221,6 +225,15 @@ def __discoverCredentialsToUse(self):
       else:
         self.kwargs[self.KW_SKIP_CA_CHECK] = skipCACheck()
 
+    # Use tokens?
+    if self.KW_USE_ACCESS_TOKEN in self.kwargs:
```

Comment:
I don't quite understand the logic here between token and server certificate

Reply:
so you're right, I'll fix it